api/src/main/java/org/pmiops/workbench/cohorts/CohortMaterializationService.java

```diff
@@ -178,74 +174,33 @@ private ParticipantCriteria getParticipantCriteria(List<CohortStatus> statusFilt
     }
   }
 
+
   private void addFilterOnConcepts(TableQuery tableQuery, Set<Long> conceptIds,
       TableConfig tableConfig) {
-    String standardConceptColumn = null;
     String sourceConceptColumn = null;
     for (ColumnConfig columnConfig : tableConfig.columns) {
-      if (DOMAIN_CONCEPT_STANDARD.equals(columnConfig.domainConcept)) {
```

Comment:
Note: this would have us looking in just source concept ID column when matching concepts.... which is probably not right for standard concepts. We may need an or here (without necessarily bothering with looking up the concepts in the database as we were before.) Discussing with Peter now.

Reply:
After further discussion, reverted the functional changes, but refactored out the logic for getting concept columns and classified concept IDs for use in ConceptBigQueryService.